### DISCUSSION TITLE
Podman needs extra capabilities (NET_RAW and MKNOD)

### DISCUSSION THREAD

**Discussion by viktomas** [haugene/docker-transmission-openvpn#2474] [General]
Hello, first, thank you to `haugene` and the community for this project. Saved me a lot of time.

The `docker run` instructions specify the `--cap-add=NET_ADMIN` capability so the container can create the tunnel.

With `podman` the following capabilities are needed:

- `NET_ADMIN` and `MKNOD` to create the tunnel
- `NET_RAW` because the container runs the `ping` command somewhere in the setup phase

so the argument for `podman` is `--cap-add NET_ADMIN,NET_RAW,MKNOD`.

I don't see many people using `podman` here, but if people would be interested, I can create a PR to add this info to the documentation. It was the only thing I had to spend time figuring out, rest of the project worked out-of-the-box 🎉 🙇

**Comment by pkishino** [comment]
Feel free to add it to the documentation under tips perhaps?
Thanks

**Reply by viktomas** [reply]
Thanks :) created a small PR https://github.com/haugene/docker-transmission-openvpn/pull/2475
